packages/arb-avm-cpp/avm_values/include/avm_values/opcodes.hpp

```diff
@@ -487,12 +487,12 @@ const std::unordered_map<OpCode, uint64_t> InstructionArbGasCost = {
     {OpCode::DEBUG_PRINT, 1},
 
     {OpCode::NEW_BUFFER, 1},
-    {OpCode::GET_BUFFER8, 10},
-    {OpCode::GET_BUFFER64, 10},
-    {OpCode::GET_BUFFER256, 10},
-    {OpCode::SET_BUFFER8, 100},
-    {OpCode::SET_BUFFER64, 100},
-    {OpCode::SET_BUFFER256, 100},
+    {OpCode::GET_BUFFER8, 3},
```

Comment:
Best to leave a comment here, explaining the logic behind how gas costs of getBuffer ops were determined.  Either they should depend on the set of the buffer (reflecting the need to walk a tree), or set to a constant equal to the cost with a maximum-size tree.